fetools-app/src/components/SearchField.jsx

```diff
@@ -10,58 +10,54 @@ const SearchField = ({ placeholderText, links, search, clearInput }) => {
 
   useEffect(() => {
     if (clearInput) {
-      setInputValue("");
-      search("");
+      setInputValue('');
+      search('');
     }
   }, [clearInput]);
 
-
   const handleKeyDown = (e) => {
-    if (e.key === "Enter") {
+    if (e.key === 'Enter') {
       search(inputValue);
     }
     if (
-      e.key === "ArrowLeft" ||
-      e.key === "ArrowRight" ||
-      e.key === "ArrowUp" ||
-      e.key === "ArrowDown" ||
-      e.key === "Backspace"
+      e.key === 'ArrowLeft' ||
+      e.key === 'ArrowRight' ||
+      e.key === 'ArrowUp' ||
+      e.key === 'ArrowDown' ||
+      e.key === 'Backspace'
     ) {
       return;
     }
     search(inputValue);
   };
 
   return (
-    <div className="flex justify-end">
+    <div className='flex justify-end relative'>
+      <label className='relative'>
       <input
-        type="text"
+        type='text'
         placeholder={placeholderText}
         value={inputValue}
         onChange={handleInputChange}
         onKeyDown={handleKeyDown}
-        className="bg-gray-100 p-3 text-lg"
+        className='bg-gray-100 p-3 text-lg pr-12 focus:ring focus:ring-[#663399] focus:outline-none'
```

Comment:
Let's add a hint of rounding here.

```suggestion
        className='bg-gray-100 p-3 text-lg pr-12 rounding-[.25rem] focus:ring focus:ring-[#663399] focus:outline-none'
```